frontend/src/components/browser/inspectorBrowser.js

```diff
@@ -189,13 +194,161 @@ class InspectorBrowser extends Component {
+                <td>
+                  {this.fetchObjectInDict(
+                    this.props.cameras,
+                    clip.camera,
+                    "name"
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <td>Folder</td>
+                <td>
+                  {this.fetchObjectInDict(
+                    this.props.folders,
+                    clip.folder,
+                    "absolutePath"
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <td>Format</td>
+                <td>{clip.format ? clip.format : "cant find format"}</td>
+              </tr>
+              {/* Fetch the resolution object from filter based on the clip:s resolution id.*/}
+              <tr>
+                <td>Resolution</td>
+                <td>
+                  {this.props.filter[clip.resolution]
+                    ? this.props.filter[clip.resolution].height +
+                      " x " +
+                      this.props.filter[clip.resolution].width
+                    : "cant find resoluton"}
+                </td>
+              </tr>
+              <tr>
+                <td>Frame rate</td>
+                <td>
+                  {clip.frame_rate ? clip.frame_rate : "cant find frame rate"}
+                </td>
+              </tr>
+              <tr>
+                <td>Playable</td>
+                <td>{clip.playable ? "Yes" : "No"}</td>
+              </tr>
+              <tr>
+                <td>Overlapping</td>
+                <td>{clip.overlapping != [] ? "Yes" : "No"}</td>
+              </tr>
+              <tr>
+                <td>Nr. of dublicates</td>
+                <td>
+                  {clip.duplicates
+                    ? clip.duplicates.length
+                    : "cant find duplicates"}
+                </td>
+              </tr>
+              <tr>
+                <td>Start Date</td>
+                <td>
+                  {clip.startTime.toDateString()
+                    ? clip.startTime.toDateString()
+                    : "cant find date"}
+                </td>
+              </tr>
+              <tr>
+                <td>Start Time</td>
+                <td>
+                  {clip.startTime.toTimeString().slice(0, 8)
+                    ? clip.startTime.toTimeString().slice(0, 8)
+                    : "cant find time"}
+                </td>
+              </tr>
+              <tr>
+                <td>End Date</td>
+                <td>
+                  {clip.endTime.toDateString()
+                    ? clip.endTime.toDateString()
+                    : "cant find date"}
+                </td>
+              </tr>
+              <tr>
+                <td>End Time</td>
+                <td>
+                  {clip.endTime.toTimeString().slice(0, 8)
+                    ? clip.endTime.toTimeString().slice(0, 8)
+                    : "cant find time"}
+                </td>
```

Comment:
Change to "Can't find X" or "Can not find X". You can also remove this if you wan't because this fields should never be undefined.